public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/EmailSharingConfiguration.tsx

```diff
@@ -78,27 +88,27 @@ const EmailList = ({
                   type="button"
                   variant="destructive"
                   fill="text"
-                  aria-label="Revoke"
-                  title="Revoke"
+                  aria-label={translatedRevokeLabel}
```

Comment:
As in completely removing the` aria-label`? There are a couple of other places that also does this too if we want to go down this route. I also think aria-label is redundant since we have the title

Reply:
Yes, I think the same too. The `aria-label` should be required for accessibility only when there is no viewable element. If we remove it we can put the translations directly in the code instead of creating a variable.